academic_observatory_workflows/config.py

```diff
@@ -35,19 +36,59 @@ def test_fixtures_folder(*subdirs) -> str:
     return os.path.join(base_path, *subdirs)
 
 
-def schema_folder() -> str:
+def workflow_test_fixtures_path(workflow_module: str, *subdirs) -> str:
+    """Get the path to the Academic Observatory Workflows test data directory.
+
+    :param workflow_module: Optional, name of the workflow. Only to be included if the schema for the workflow is in
```

Comment:
As it stands, *workflow_module* is not optional. We could consolidate this function and *test_fixtures_folder()* in the same manner as the other functions such that it is optional. Unless we expect that the root level fixtures folder will soon become obsolete?

Reply:
Thanks Keegan, I've done this, `workflow_module` has to be a keyword argument after `*subdirs` if it is optional:

```python
def test_fixtures_folder(*subdirs: str, workflow_module: Optional[str] = None) -> str:
```